CHANGELOG.rst

```diff
@@ -14,6 +14,10 @@ Change Log
 Unreleased
 ~~~~~~~~~~
 
+[1.0.2] - 2020-09-14
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+* Move to Apache License
+
 [1.0.1] - 2020-09-14
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 * Minor updates to attrs, code-annotations, and freezegun dependencies
```

Comment:
I suspect we should move these

Reply:
not sure what you mean...

Reply:
I mistook the section levels & thought these were under "Unreleased". Empty sections mess with me.